bottle.py

```diff
@@ -543,7 +544,17 @@ def get_undecorated_callback(self):
         func = getattr(func, '__func__' if py3k else 'im_func', func)
         closure_attr = '__closure__' if py3k else 'func_closure'
         while hasattr(func, closure_attr) and getattr(func, closure_attr):
-            func = getattr(func, closure_attr)[0].cell_contents
+            attributes = getattr(func, closure_attr)
+            func = attributes[0].cell_contents
+
+            # in case of decorators with multiple arguments
+            if not isinstance(func, FunctionType) and func is not None:
```

Comment:
What case is` func is not None` covering here, and why is it not covered in the following `filter` code?